Components/grid/grid.css

```diff
@@ -0,0 +1,125 @@
+@import url("../../main.css");
+@import url('../../colors.css');
+@import url('../../typography.css');
+
+
+/* colors for unit borders */
+:root {
+    --color1: #001219;
+    --color2: #005f73;
+    --color3: #3d9397;
+    --color4: #94d28d;
+    --color5: #e9d8a6;
+    --color6: #ee9800;
+    --color7: #ca6702;
+    --color8: #bb3e03;
+    --color9: #ae2012;
+    --color10: #9b2226;
+    --color11: rgba(255, 255, 255, 0.8);
```

Comment:
A suggestion here would be that you could give them proper names. Just reading color1, color2 I'm not able to understand basically which shade of color is it.
You could use red, green, blue, yellow and all or else primary, secondary, error, success, warning, etc.

Reply:
haha yes, I was lazy here as I have written many CSS variables for colors in "colors.css" and I have to come up with unique names for these ones as well 😂, anyways I will try to come up with some names and name these as well 👍🏼